src/components/events/DetailsAddon.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <DetailsBase class="details-addon">
+    <template #header>
+      ADDONS
+    </template>
+    <template #content>
+      <div
+        class="event-details__content-block"
+        v-for="(value, key) in addons"
+        :key="key"
+      >
+        <div class="event-details__key">
+          {{ key }}
+        </div>
+        <div class="event-details__value">
+          <JsonViewer
```

Comment:
why the  `json viewer` is used here?

Reply:
@neSpecc i think that addons keys values can be a object like window value

Reply:
addons should be shown without json collapsing, as a separated keys. Also, you probably will need a separated components for each language, because adddons can be different